developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_27.rst

```diff
@@ -38,19 +45,36 @@ Nextcloud 27 no longer loads classes in the deprecated :ref:`PSR-0 naming standa
 Added APIs
 ^^^^^^^^^^
 
-* tbd
-
-Changed APIs
-^^^^^^^^^^^^
+* ``\OCP\AppFramework\Utility\ITimeFactory`` is now a ``\PSR\Clock\ClockInterface`` following the `PSR-20 standard <https://www.php-fig.org/psr/psr-20/#21-clockinterface>`_. (`nextcloud/server#35872 <https://github.com/nextcloud/server/pull/35872>`_)
+* ``\OCP\Accounts\IAccountManager::PROPERTY_DISPLAYNAME_LEGACY`` was added to keep the provisioning API compatible. (`nextcloud/server#36665 <https://github.com/nextcloud/server/pull/36665>`_)
+* A new system config ``ratelimit.protection.enabled`` (boolean, default true) was added to allow developers to **disable** rate limiting. It is **not** recommended to disable this on a production/live system. (`nextcloud/server#37542 <https://github.com/nextcloud/server/pull/37542>`_)
+* A new OCP API under ``\OCP\SpeechToText`` was introduced to allow automatic transcription of media (`nextcloud/server#37674 <https://github.com/nextcloud/server/pull/37674>`_)
+* Added a new interface ``\OCP\BackgroundJob\IParallelAwareJob`` that ``\OCP\BackgroundJob\Job`` now implements. It can be used to allow specifying that multiple instances of a job should not be run at the same time. Also added ``\OCP\BackgroundJob\IJobList#hasReservedJob(?string $className = null)`` method to check the condition (`nextcloud/server#37835 <https://github.com/nextcloud/server/pull/37835>`_)
+* New property ``$actionLabel`` has been added to the ``\OCP\Files\Template\TemplateFileCreator`` class with a respective setter ``TemplateFileCreator::setActionLabel`` and getter ``TemplateFileCreator::getActionLabel``.  (`nextcloud/server#37929 <https://github.com/nextcloud/server/pull/37929>`_ + `nextcloud/server#37955 <https://github.com/nextcloud/server/pull/37955>`_)
+* A new interface ``\OCP\Group\Backend\ISearchableGroupBackend`` was added for group backends supporting new method ``searchInGroup`` to search among a group users in an efficient way. (`nextcloud/server#32866 <https://github.com/nextcloud/server/pull/32866>`_)
 
-* tbd
 
 Deprecated APIs
 ^^^^^^^^^^^^^^^
 
-* tbd
+* ``\OCP\AppFramework\Utility\ITimeFactory::getTime()`` and ``\OCP\AppFramework\Utility\ITimeFactory::getDateTime()`` were deprecated, because the interface is now a  is now a ``\PSR\Clock\ClockInterface`` following the `PSR-20standard <https://www.php-fig.org/psr/psr-20/#21-clockinterface>`_. (`nextcloud/server#35872 <https://github.com/nextcloud/server/pull/35872>`_)
```

Comment:
```suggestion
* ``\OCP\AppFramework\Utility\ITimeFactory::getTime()`` and ``\OCP\AppFramework\Utility\ITimeFactory::getDateTime()`` were deprecated, because the interface is now a ``\PSR\Clock\ClockInterface`` following the `PSR-20standard <https://www.php-fig.org/psr/psr-20/#21-clockinterface>`_. (`nextcloud/server#35872 <https://github.com/nextcloud/server/pull/35872>`_)
```